app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,11 @@
+child :content_facet => :content_attributes do
```

Comment:
I'm not necessarily a fan of *_attributes either but i figured 'content' and 'subscriptions' may be confusing (especially subscriptions as you may expect a list of subscriptions).  Another option would just be 'content_facet' and 'subscription_facet'.

I'm open to suggestions :)